leptos_hotkeys/src/context.rs

```diff
@@ -38,8 +44,7 @@ where
     });
 
     #[cfg(not(feature = "ssr"))]
-    let pressed_keys: RwSignal<std::collections::HashMap<String, web_sys::KeyboardEvent>> =
-        RwSignal::new(std::collections::HashMap::new());
+    let pressed_keys: RwSignal<KeyPresses> = RwSignal::new(KeyPresses::default());
```

Comment:
Since we renamed the inner `HotkeysContext` `pressed_keys` to `keys_pressed`. Can we also rename all signals that follow this convention as well?